api/client/run.go

```diff
@@ -6,9 +6,11 @@ import (
 	"net/url"
 	"os"
 	"runtime"
+	"strings"
 
 	"github.com/Sirupsen/logrus"
 	Cli "github.com/docker/docker/cli"
+	"github.com/docker/docker/errors"
```

Comment:
Just for consistency, please alias this to "derr". since "errors" is used in other files for go's errors package.